integrations/kubernetes.md

```diff
@@ -16,9 +18,7 @@ This guide explains how to connect a Kubernetes cluster to $CLOUD_LONG, configur
 ## Prerequisites
 
 <IntegrationPrereqs />
```

Comment:
Don't think you need to install self-hosted before you install self-hosted on kubernetes. Can you just add the "To follow the steps on this page:" above the kubernetes pre-requisites please.